ingeniamotion/wizard_tests/phasing_check.py

```diff
@@ -165,7 +167,7 @@ def not_forced_phasing(self) -> None:
         self.mc.motion.wait_for_velocity(0, servo=self.servo, axis=self.axis)
 
     @BaseTest.stoppable
-    def check_motor_commutation(self) -> ResultType:
+    def __check_motor_commutation(self) -> ResultType:
```

Comment:
Have you checked that all this modules that you are converting to private are not being used on ml3?

Reply:
Yes, I did. They are not used.